### ISSUE TITLE
lsposed can't operate on RMX3562's newest system version[Bug] 

### ISSUE THREAD

**Issue by guiyinrenshi** [LSPosed/LSPosed#2659]
### Steps to reproduce/复现步骤

1.在magisk manager安装zygisk版本的lsposed压缩包
2.升级系统至RMX3562_13.1.0.210(CN01)
2.重启手机
3.点击lsposed 管理器

### Expected behaviour/预期行为

能够正常打开lsposed管理器且可以使用依托lsposed管理器的所有应用

### Actual behaviour/实际行为

无法打开lsposed管理器且无法使用依托lsposed管理器的所有应用

### Xposed Module List/Xposed 模块列表

```shell
scene5
luckytool
Don't kill
隐藏应用列表
网抑刁毛云
```

### Magisk Module List/Magisk 模块列表

```shell
A-ColorOS设备与电量使用详情lite
A-SOUI Games Optimization
A1-内存管理
AD hosts(国内版)
Busybox for Android NDK
C++Busybox
Disable HW Overlay
MAT-目录定向挂载
Scene的附加模块
Scene附加模块（2）修改版
shamiko
UnityFix
zygisk-LSPosed
zygisk-指纹支付系列
屏蔽数字人民币检测root
爱玩机工具箱（应用核心功能拓展）
移除温控【正式版】
自动神仙救砖
音量键极速救砖
```

### LSPosed version/LSPosed 版本

v1.8.6(6712)

### Android version/Android 版本

13

### Magisk version/Magisk 版本

26.1(26100)

### Riru version/Riru 版本

N/A

### Version requirement/版本要求

- [x] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

[bugreport-RMX3562-TP1A.220905.001-2023-08-03-15-02-26.zip](https://github.com/LSPosed/LSPosed/files/12248176/bugreport-RMX3562-TP1A.220905.001-2023-08-03-15-02-26.zip)

**Comment by yujincheng08** [comment]
Fixed by https://github.com/LSPosed/LSPosed/commit/8b3e331c5e1b4cf303d83d3364b5cd2437a44ca2

**Comment by guiyinrenshi** [comment]
> Fixed by [8b3e331](https://github.com/LSPosed/LSPosed/commit/8b3e331c5e1b4cf303d83d3364b5cd2437a44ca2)

if you have New version link

**Comment by guiyinrenshi** [comment]
> Fixed by [8b3e331](https://github.com/LSPosed/LSPosed/commit/8b3e331c5e1b4cf303d83d3364b5cd2437a44ca2)

if you have New version link

**Comment by Yangleis** [comment]
> > Fixed by [8b3e331](https://github.com/LSPosed/LSPosed/commit/8b3e331c5e1b4cf303d83d3364b5cd2437a44ca2)
> 
> if you have New version link

在Action页面下载自动构筑的版本。
